src/psbt.c

```diff
@@ -48,23 +48,27 @@
 #define PSBT_OUT_BIP32_DERIVATION 0x02
 
 #ifdef BUILD_ELEMENTS
-#define PSET_IN_VALUE 0x00
-#define PSET_IN_VALUE_BLINDER 0x01
-#define PSET_IN_ASSET 0x02
-#define PSET_IN_ASSET_BLINDER 0x03
+#define PSET_GLOBAL_SCALRA 0x00
```

Comment:
```suggestion
#define PSET_GLOBAL_SCALAR 0x00
```
extremely drive by nit lol